scripts/run_backend_tests.py

```diff
@@ -603,7 +603,43 @@ def main(args: Optional[List[str]] = None) -> None:
     if parsed_args.generate_coverage_report:
         subprocess.check_call([sys.executable, '-m', 'coverage', 'combine'])
         report_stdout, coverage = check_coverage(True)
+        print('')
+        print(
+            '+-------------------------------------------------------------'
+            '--------------------------------------------------------------'
+            '-------+'
```

Comment:
Please limit your output lines to 80 characters in length so that they stay readable on narrow terminal windows (e.g. when developers have a terminal side-by-side with their code editor)

Reply:
ok @U8NWXD i will make it only 80 character long line.